flixel/FlxGame.hx

```diff
@@ -379,8 +344,6 @@ class FlxGame extends Sprite
 
 		// make sure the cursor etc are properly scaled from the start
 		resizeGame(FlxG.stage.stageWidth, FlxG.stage.stageHeight);
-
-		Assets.addEventListener(Event.CHANGE, FlxG.bitmap.onAssetsReload);
```

Comment:
are we sure this stuff isn't ever used?

Reply:
```haxe
Assets.addEventListener(Event.CHANGE, FlxG.bitmap.onAssetsReload);
```

This essentially listens for the `lime.utils.Assets.onChange` event, which is dispatched whenever a library that has been registered dispatches its own `onChange` event. However, I can't find any instance of code that would dispatch `AssetLibrary.onChange` at all, so it seems to me like this is an unused feature.

Here are the search results for `onChange` in Lime: https://github.com/search?q=repo%3Aopenfl%2Flime%20onChange&type=code
OpenFL does not dispatch it either: https://github.com/search?q=repo%3Aopenfl%2Fopenfl+onChange&type=code

Reply:
it might be worth going the the openfl forum or discord channel and asking, just to be sure

Reply:
Got a reply on the Discord channel from Josh Tynjala, quote:

> perhaps it is to allow someone to subclass `AssetLibrary` and dispatch `onChange` manually.

So you were right, it seems like this should actually stay. However, `FlxGraphic.onAssetsReload` made use of `undump` to refresh the graphic.

To be clear on what `undump` does, it refreshes the bitmap if it has an `assetsKey` or `assetsClass`, so it still does something even if it wasn't previously dumped. Maybe we should bring it back but rename it appropiately like `refreshBitmap`, or just keep the code inside `onAssetsReload`?